{{cookiecutter.repo_name}}/.travis.yml

```diff
@@ -20,17 +23,26 @@ before_install:
   - lsb_release -a
 install:
   - pip install tox
+    {%- if cookiecutter.test_runner|lower == "pytest" %} pytest-travis-fold{% endif %}
```

Comment:
This needs to be in `tox.ini`. Tox doesn't create virtualenvs with `--system-site-packages` option. It's a bit unfortunate I know but pytest-travis-fold doesn't do anything if you're not on travis right?